fmriprep/cli/run.py

```diff
@@ -701,7 +701,7 @@ def parse_spaces(opts):
 
     if opts.cifti_output:
         # CIFTI grayordinates to corresponding FSL-MNI resolutions.
-        vol_res = '32'[opts.cifti_output == '91k']
+        vol_res = '62'[opts.cifti_output == '91k']
```

Comment:
not sure if we should even differentiate this here, in current master we just add `res2` and then use the resampling check within the `GenCifti` interface